dts/bindings/dai/nxp,dai-sai.yaml

```diff
@@ -84,3 +84,28 @@ properties:
       If this property is not specified, the receiver will be set to ASYNC.
       If one side is SYNC then the other MUST be ASYNC. Failing to meet this
       condition will result in a failed BUILD_ASSERT().
+  tx-dline:
+    type: int
```

Comment:
tx-dataline would be more clear and there are only 3 extra chars.

Reply:
renamed properties to `tx_dataline` and `rx_dataline`